### ISSUE TITLE
Nested dirname calls inspection shows error when levels is set

### ISSUE THREAD

**Issue by tarlepp** [kalessil/phpinspectionsea#572]
Screenshot shows that all is ok, but inspection is still complaining about this.
![virtualbox_code 2017_07_10_2017_16_24_48](https://user-images.githubusercontent.com/595561/31308261-4ab6c41a-ab7c-11e7-9798-e1aa40fa4569.png)

```php
$symfonyRequirements = new SymfonyRequirements(dirname(realpath($autoloader), 2));
```

**Comment by kalessil** [comment]
Thank you @tarlepp for reporting. Looks weird, I'll try to reproduce.

**Comment by kalessil** [comment]
@tarlepp : confirmed, working on it.
